### PR TITLE
Feature/extract macros

### PR DESCRIPTION
**Pull request by robertparkinson**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
*
*
*

### CODE REVIEW

File: src/main/app/routes/what-type-of-claimant.ts [hunk 1]
```diff
@@ -2,21 +2,26 @@ import * as express from 'express'
...
 
     if (!form.hasErrors()) {
-      res.locals.user.claimDraft.claimantType = form.model
-      res.redirect('what-type-of-claimant')
+      res.locals.user.claimDraft.claimant.claimantType = form.model
+      ClaimDraftMiddleware.save(res, next)
+        .then(() => {
+          res.redirect('/claim/start')
+        })
+      res.redirect('/claim/what-type-of-claimant')
```
Comment (rlewan) [inline src/main/app/routes/what-type-of-claimant.ts]: Both redirects will execute because promise is async.

Comment (robertparkinson) [inline src/main/app/routes/what-type-of-claimant.ts]: removed the second redirect

File: src/main/app/drafts/models/claimant.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { ClaimantType } from '../../forms/models/claimantType'
```
Comment (rlewan) [inline src/main/app/drafts/models/claimant.ts]: Please don't use a relative path here, access it via one of the mappings from ts config.

Comment (robertparkinson) [inline src/main/app/drafts/models/claimant.ts]: tsconfig was only added as part of the other PR. I can update it in that PR

Comment (robertparkinson) [inline src/main/app/drafts/models/claimant.ts]: ok merged the other PR and removed the relative path

**Pull request merged**